src/adaptation/stepsize.jl

```diff
@@ -8,7 +8,7 @@ mutable struct DAState{T<:AbstractScalarOrVec{<:AbstractFloat}}
     H_bar::T
 end
 
-computeμ(ϵ::AbstractScalarOrVec{<:AbstractFloat}) = log.(10 * ϵ)
+computeμ(ϵ::AbstractFloat) = log(10 * ϵ)
```

Comment:
Caution is required here: these support the vectorised version of `HMC`. Do you know how `map` would differ from broadcasting here?

Reply:
The results of the calculations won't be affected by this change, but using the non-broadcasted formulation for scalars and `map` for vectors of floats will remove the broadcasting overhead and reduce stress on the compiler, i.e., generally it reduces compilation time. Sometimes it also helps type inference (but this case is too simple for this effect I assume).

In my experience, broadcasting is useful if one's actually broadcasting values of different size and dimensions but otherwise often a suboptimal choice.